tests/test_cmdline.py

```diff
@@ -392,6 +409,25 @@ def check_func(data):
                           expected_output_file="result.json",
                           success=False)
 
+    @skipIf(importlib.util.find_spec("trio") is None, reason="Trio-specific test")
+    def test_log_trio(self):
+        def check_func(data):
+            tids = set()
+            for entry in data["traceEvents"]:
+                tids.add(entry["tid"])
+            self.assertGreaterEqual(len(tids), 4)
+
+        if int(platform.python_version_tuple()[1]) >= 7:
```

Comment:
The test is good. I realized I used ``platform.python_version_tuple`` in another test(which was written a long time ago). I believe ``sys.version_info`` is preferred, at least by ``mypy``, and clearer. Could you change this line to ``sys.version_info >= (3, 7)``?